packages/components/ng-at-internet/src/directive.js

```diff
@@ -69,7 +69,66 @@ export /* @ngInject */ function trackOnDirective(atInternet) {
   };
 }
 
+/**
+ * @ngdoc directive
+ * @require atInternetProvider
+ * @name trackImpression
+ * @description
+ * Simple attribute directive to track impressions on DOM elements.
+ * https://developers.atinternet-solutions.com/javascript-fr/campagnes-javascript-fr/publicites-on-site-javascript-fr/
```

Comment:
We should probably refer to the english documentation https://developers.atinternet-solutions.com/javascript-en/campaigns-javascript-en/on-site-ads-javascript-en/